src/pages/IssueNft/components/IssueNftForm.tsx

```diff
@@ -0,0 +1,209 @@
+import classNames from 'classnames';
+import Select from 'react-select';
+import { Button, MxLink } from 'components';
+import { DataTestIdsEnum } from 'localConstants';
+import { routeNames } from 'routes';
+import { NftEnumType } from 'types';
+import { useIssueNftForm } from '../hooks';
+import { IssueNftFieldsEnum } from '../types';
+
+export const IssueNftForm = () => {
+  const { formik, isLoading, collections, selectedCollection } =
+    useIssueNftForm();
+
+  return (
+    <form
+      onSubmit={formik.handleSubmit}
+      noValidate
+      className='d-flex flex-column'
+    >
+      <div className='flex flex-col gap-4 h-full'>
+        <div className='flex flex-row align-middle w-full'>
+          <div className='flex flex-col w-full'>
+            <Select
+              className='text-sm text-gray-700 placeholder-gray-400 w-full'
+              isLoading={isLoading}
+              options={collections as any}
+              name={IssueNftFieldsEnum.collection}
+              onChange={(option) =>
+                formik.setFieldValue(IssueNftFieldsEnum.collection, option)
+              }
+              onBlur={() =>
+                formik.setFieldTouched(IssueNftFieldsEnum.collection, true)
+              }
+              value={formik.values[IssueNftFieldsEnum.collection]}
+            />
+            {formik.touched[IssueNftFieldsEnum.collection] &&
+              formik.errors[IssueNftFieldsEnum.collection] && (
+                <div
+                  className='text-red-600 text-sm mt-1'
+                  data-testid={DataTestIdsEnum.collectionError}
+                >
+                  {formik.errors[IssueNftFieldsEnum.collection] as any}
```

Comment:
Why `any`? Can we declare a proper type or perhaps remove the casting entirely?

Reply:
Because the field IssueNftFieldsEnum.collection in formmik is an object. TS thinks formik.errors[IssueNftFieldsEnum.collection] is also an object, but it is just a string